.github/ISSUE_TEMPLATE/report-package-problems.yml

```diff
@@ -2,23 +2,36 @@
 name: Packaging problem report
 description: Report problems with a package. This includes build failures or other issues related to a package. This is *not* for bugs with installed software.
 labels: ["Package Problem", "Priority: Normal"]
```

Comment:
If we want to use new label it should be "Packaging Problem" instead of "Package Problem". We can also use already available label "Does not build".

Reply:
Good suggestion, changed.

Reply:
That works for me. I've asked in Packaging to get community input and there was no one opposed. Ermo has ideas on further improvements to the labels with parent:child formatting, but that can be a separate effort in a separate PR.

I've updated the label.